File: Php/app/Urls/HomegameEditUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class HomegameEditUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::homegameEdit, $homegame);
		}

	}

}

File: Web/Models/Url/HomegameEditUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.Url{

	public class HomegameEditUrlModel : HomegameUrlModel{

		public HomegameEditUrlModel(Homegame homegame) : base(RouteFormats.HomegameEdit, homegame)
        {
		}

	}

}